20:35:28 29.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
20:35:28 29.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
20:35:28 29.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
20:35:28 29.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
20:35:29 29.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
20:35:29 29.01.2016 WARN  JdbcServicesImpl  configure:204 - HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (Access denied for user 'root'@'localhost' (using password: YES))
20:35:29 29.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
20:35:29 29.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:88 - HHH000422: Disabling contextual LOB creation as connection was null
20:35:29 29.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
20:40:23 29.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
20:40:23 29.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
20:40:23 29.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
20:40:23 29.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
20:40:24 29.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
20:40:24 29.01.2016 WARN  JdbcServicesImpl  configure:204 - HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (Access denied for user 'root'@'localhost' (using password: YES))
20:40:24 29.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
20:40:24 29.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:88 - HHH000422: Disabling contextual LOB creation as connection was null
20:40:24 29.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
13:08:43 30.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
13:08:44 30.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
13:08:44 30.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
13:08:44 30.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
13:08:44 30.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
13:08:44 30.01.2016 WARN  JdbcServicesImpl  configure:204 - HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (Access denied for user 'root'@'localhost' (using password: YES))
13:08:44 30.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
13:08:44 30.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:88 - HHH000422: Disabling contextual LOB creation as connection was null
13:08:45 30.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
11:40:21 31.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
11:40:22 31.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
11:40:22 31.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
11:40:22 31.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
11:40:22 31.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
11:40:22 31.01.2016 WARN  JdbcServicesImpl  configure:204 - HHH000342: Could not obtain connection to query metadata : Cannot create PoolableConnectionFactory (Access denied for user 'root'@'localhost' (using password: YES))
11:40:22 31.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
11:40:22 31.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:88 - HHH000422: Disabling contextual LOB creation as connection was null
11:40:22 31.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
12:39:44 31.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
12:39:44 31.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
12:39:44 31.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
12:39:44 31.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
12:39:44 31.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
12:39:45 31.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
12:39:45 31.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:97 - HHH000423: Disabling contextual LOB creation as JDBC driver reported JDBC version [3] less than 4
12:39:45 31.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
13:16:46 31.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
13:16:46 31.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
13:16:46 31.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
13:16:46 31.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
13:16:46 31.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
13:16:47 31.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
13:16:47 31.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:97 - HHH000423: Disabling contextual LOB creation as JDBC driver reported JDBC version [3] less than 4
13:16:47 31.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
16:38:42 31.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
16:38:42 31.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
16:38:42 31.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
16:38:42 31.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
16:38:42 31.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
16:38:43 31.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
16:38:43 31.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:97 - HHH000423: Disabling contextual LOB creation as JDBC driver reported JDBC version [3] less than 4
16:38:43 31.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
16:41:22 31.01.2016 INFO  LogHelper  logPersistenceUnitInformation:46 - HHH000204: Processing PersistenceUnitInfo [
	name: primary
	...]
16:41:22 31.01.2016 INFO  Version  logVersion:54 - HHH000412: Hibernate Core {4.3.6.Final}
16:41:22 31.01.2016 INFO  Environment  <clinit>:239 - HHH000206: hibernate.properties not found
16:41:22 31.01.2016 INFO  Environment  buildBytecodeProvider:346 - HHH000021: Bytecode provider name : javassist
16:41:22 31.01.2016 INFO  Version  <clinit>:66 - HCANN000001: Hibernate Commons Annotations {4.0.5.Final}
16:41:23 31.01.2016 INFO  Dialect  <init>:145 - HHH000400: Using dialect: org.hibernate.dialect.MySQLDialect
16:41:23 31.01.2016 INFO  LobCreatorBuilder  useContextualLobCreation:97 - HHH000423: Disabling contextual LOB creation as JDBC driver reported JDBC version [3] less than 4
16:41:23 31.01.2016 INFO  ASTQueryTranslatorFactory  <init>:47 - HHH000397: Using ASTQueryTranslatorFactory
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/AuthorHome!com.softserveinc.booklibrary.session.persist.home.impl.AuthorHomeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/AuthorFacade!com.softserveinc.booklibrary.session.persist.facade.impl.AuthorFacadeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/BookHome!com.softserveinc.booklibrary.session.persist.home.impl.BookHomeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/BookFacade!com.softserveinc.booklibrary.session.persist.facade.impl.BookFacadeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/ReviewHome!com.softserveinc.booklibrary.session.persist.home.impl.ReviewHomeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:24 31.01.2016 ERROR TestHelper  getRemoteBean:79 - Remote bean  could not found
javax.naming.NameNotFoundException: Name "global/BookLibrary-cdi/ReviewFacade!com.softserveinc.booklibrary.session.persist.facade.impl.ReviewFacadeRemote" not found.
	at org.apache.openejb.core.ivm.naming.IvmContext.federate(IvmContext.java:199)
	at org.apache.openejb.core.ivm.naming.IvmContext.lookup(IvmContext.java:151)
	at org.apache.xbean.naming.context.ContextFlyweight.lookup(ContextFlyweight.java:55)
	at com.softserveinc.model.util.TestHelper.getRemoteBean(TestHelper.java:76)
	at com.softserveinc.model.util.BaseTest.getRemoteBeans(BaseTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
16:41:25 31.01.2016 WARN  AbstractTableMetaData  getDataTypeFactory:166 - Potential problem found: The configured data type factory 'class org.dbunit.dataset.datatype.DefaultDataTypeFactory' might cause problems with the current database 'MySQL' (e.g. some datatypes may not be supported properly). In rare cases you might see this message because the list of supported database products is incomplete (list=[derby]). If so please request a java-class update via the forums.If you are using your own IDataTypeFactory extending DefaultDataTypeFactory, ensure that you override getValidDbProducts() to specify the supported database products.
16:41:25 31.01.2016 WARN  AbstractTableMetaData  getDataTypeFactory:166 - Potential problem found: The configured data type factory 'class org.dbunit.dataset.datatype.DefaultDataTypeFactory' might cause problems with the current database 'MySQL' (e.g. some datatypes may not be supported properly). In rare cases you might see this message because the list of supported database products is incomplete (list=[derby]). If so please request a java-class update via the forums.If you are using your own IDataTypeFactory extending DefaultDataTypeFactory, ensure that you override getValidDbProducts() to specify the supported database products.
16:41:27 31.01.2016 WARN  AbstractTableMetaData  getDataTypeFactory:166 - Potential problem found: The configured data type factory 'class org.dbunit.dataset.datatype.DefaultDataTypeFactory' might cause problems with the current database 'MySQL' (e.g. some datatypes may not be supported properly). In rare cases you might see this message because the list of supported database products is incomplete (list=[derby]). If so please request a java-class update via the forums.If you are using your own IDataTypeFactory extending DefaultDataTypeFactory, ensure that you override getValidDbProducts() to specify the supported database products.
16:41:30 31.01.2016 WARN  AbstractTableMetaData  getDataTypeFactory:166 - Potential problem found: The configured data type factory 'class org.dbunit.dataset.datatype.DefaultDataTypeFactory' might cause problems with the current database 'MySQL' (e.g. some datatypes may not be supported properly). In rare cases you might see this message because the list of supported database products is incomplete (list=[derby]). If so please request a java-class update via the forums.If you are using your own IDataTypeFactory extending DefaultDataTypeFactory, ensure that you override getValidDbProducts() to specify the supported database products.
